cypress/support/commands.js

```
@@ -1,70 +1,22 @@
-// ***********************************************
-// This example commands.js shows you how to
-// create various custom commands and overwrite
-// existing commands.
-//
-// For more comprehensive examples of custom
-// commands please read more here:
-// https://on.cypress.io/custom-commands
-// ***********************************************
-//
-//
-// -- This is a parent command --
-// Cypress.Commands.add('login', (email, password) => { ... })
-//
-//
-// -- This is a child command --
-// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
-//
-//
-// -- This is a dual command --
-// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
-//
-//
-// -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
-
-const imgUrl = 'https://static.productionready.io/images/smiley-cyrus.jpg';
-
 Cypress.Commands.add('login', (email, username, password) => {
-  cy.request('POST', '/api/users', {
-    user: {
-      email,
-      username,
-      password
-    }
+  cy.request({
+    method: 'POST',
+    url: '/api/users',
+    body: {
+      user: { email, username, password }
+    },
+    failOnStatusCode: false, 
```

Optional but important: `failOnStatusCode: false` is used on the registration request. That prevents Cypress from failing the test when the API returns an error (e.g., user already exists). If you keep it, explicitly check `response.status` and implement logic for 200/201 vs 4xx/5xx (for example, if 422 returned because user exists, proceed to login; if other error, fail). Silently ignoring all errors can hide real problems.